spiff/models.py

```diff
@@ -18,34 +25,31 @@ def __init__(
         gnn_class: Type[basic_gnn.BasicGNN],

```

Comment:
If this is created via a factory mapping strings to classes, why bother supplying a type, not just an instance?